Riot/Modules/Room/RoomViewController.m

```diff
@@ -4614,7 +4614,13 @@ - (void)editEventContentWithId:(NSString*)eventId
 {
     MXEvent *event = [self.roomDataSource eventWithEventId:eventId];
     
-    if ([self inputToolbarConformsToToolbarViewProtocol])
+    if ([self inputToolbarConformsToHtmlToolbarViewProtocol]) {
+        self.textMessageBeforeEditing = self.inputToolbarView.attributedTextMessage;
```

Comment:
Does this have any effect? Or when you do the UI part you will do the message restoration with a html prop?

Reply:
Yeah when I will do the UI part, I'll probably reimplement this line of code, because I doubt as it is right now it has any effect (there is no cancel yet, so can't really tell)

Reply:
I'll add a TODO comment to keep track of this.